### ISSUE TITLE
[Bug] Weapon Supercharge ammo font swaps from HD to Low Res

### ISSUE THREAD

**Issue by DatMaxNub** [luciusDXL/TheForceEngine#442]
Version: 1.10

Game: Dark Forces

Issue: With the HD Hud Enhancement on from the Remaster, grabbing a Supercharge switches the ammo counter font to the low res version. This doesn't happen on the Remaster.

<img width="1280" alt="Screenshot 2024-08-25 22-22-07" src="https://github.com/user-attachments/assets/44449589-eaa8-4700-b12b-348cbe5c55e2">

**Comment by luciusDXL** [comment]
It probably can't find the HD font asset. Did you verify this works in the Remaster?

**Comment by mlauss2** [comment]
just type LARANDY and the ammo font changes to the original blocky one.

**Comment by DatMaxNub** [comment]
> @DatMaxNub TFE probably can't find the HD font asset. Did you verify this works in the Remaster?

Yes I verified that works correctly in the Remaster. The HD font shows up fine  when not supercharged

**Comment by luciusDXL** [comment]
Ok, good to know, thanks.

**Comment by mlauss2** [comment]
This is probably related to this:
```
[Error : HUD] Cannot load font 'SuperWep2.fnt'
```

**Comment by luciusDXL** [comment]
> This is probably related to this:
> 
> ```
> [Error : HUD] Cannot load font 'SuperWep2.fnt'
> ```
> 
> This notice has been there since the introduction of enhanced.gob support.
> 
> easy enough fix for this bug:
> 
> ```diff
> index ab2b4bfa..91228cad 100644
> --- a/TheForceEngine/TFE_DarkForces/hud.cpp
> +++ b/TheForceEngine/TFE_DarkForces/hud.cpp
> @@ -98,7 +98,7 @@ namespace TFE_DarkForces
>                 "ArmNum.fnt",
>                 "HelNum.fnt",
>                 "AmoNum2.fnt",
> -               "SuperWep2.fnt",
> +               "SuperWp2.fnt",
>                 "ArmNum2.fnt",
>                 "HelNum2.fnt",
>         };
> ```

Really, just a typo on my part huh... :(

Ok - I will add a fix in for the next release, thanks for spotting it @mlauss2.

**Comment by TP555** [comment]
Hi

@luciusDXL

Its not only the Longer File Names issue, TFE can't load this Font/s from the Remastered Version

I had try this pack as Mod and Copy the ex. SuperWep2.fnt ranmed to SuperWep.fnt.
But this must affect all new Fnt's with 2 !

![Font](https://github.com/user-attachments/assets/0f2f84cb-1943-4da1-88d5-953505cfcde1)

Cheers!
